### DISCUSSION TITLE
Immich not recognizing camera model from EXIF data

### DISCUSSION THREAD

**Discussion by kikendo** [immich-app/immich#11960] [Q&A]
This used to work just fine but now all I see in my options are filters for brand, the camera model always comes out as "no results".
I already re-scanned al metadata to see if it was some database problem, same result.

When you click an image, in the Info panel, the camera model displays correctly.

Server version v1.112.1

**Comment by schuhbacca** [comment]
Created issue here: https://github.com/immich-app/immich/issues/11961

**Comment by kikendo** [comment]
Nice one, I did not realize this only happened on the web interface. Definitely works on the app.
